sentry-sidekiq/lib/sentry/sidekiq-scheduler/scheduler.rb

```diff
@@ -37,7 +37,18 @@ def new_job(name, interval_type, config, schedule, options)
         # so we convert it to minutes before passing in to the monitor.
         monitor_config = case interval_type
           when "cron"
-            Sentry::Cron::MonitorConfig.from_crontab(schedule)
+            parsed_cron = ::Fugit.parse_cron(schedule)
```

Comment:
Can we mention the origin of `Fugit` in a comment? It's a second-level dependency of `sidekiq-scheduler`, through the `rufus-scheduler` gem.

Reply:
https://github.com/getsentry/sentry-ruby/pull/2215